### PR TITLE
add oca_dependencies.txt

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:  server-tools is a sub dependency

**Comment by bwrsandman** [comment]
:+1:

**Comment by gurneyalex** [comment]
hopefully the build will be green and we can merge this to fix the current build (cc @lepistone)

**Comment by lepistone** [comment]
thanks for taking care of that @gurneyalex. I'll merge as soon as it gets green.

File: .travis.yml [hunk 1]
```diff
@@ -17,14 +17,9 @@ virtualenv:
   system_site_packages: true
 
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/OCA/maintainer-quality-tools.git -b ${VERSION} ${HOME}/maintainer-quality-tools
```
Comment (guewen) [inline .travis.yml]: `-b ${VERSION}` should not  be there for `maintainer-quality-tools`

**Comment by lepistone** [comment]
:+1:

**Pull request merged**